shared/actions/profile/proofs.tsx

```diff
@@ -61,7 +61,7 @@ const recheckProof = (state, action: ProfileGen.RecheckProofPayload) =>
 // only let one of these happen at a time
 let addProofInProgress = false
 function* addProof(state, action: ProfileGen.AddProofPayload) {
-  const service = More.isPlatformsExpandedType(action.payload.platform)
+  const service = More.asPlatformsExpandedType(action.payload.platform)
```

Comment:
This can still be the `is` version right?

Reply:
no, the type guard wants a boolean. this original function is a cast to the thing or null, so i renamed the old on `as` and made the type guard `is` as i think its more accurate

Reply:
Oh right. I missed the `switch (service)` below and thought it was only used in the line immediately below. LGTM